test/unit/range/views/view_minimiser_hash_test.cpp

```diff
@@ -39,10 +39,10 @@ template <>
 struct iterator_fixture<iterator_type> : public ::testing::Test
 {
     using iterator_tag = std::forward_iterator_tag;
-    static constexpr bool const_iterable = true;
+    static constexpr bool const_iterable = false;
```

Comment:
Why does this change make the minimiser_hash non-const iterable?

Reply:
It is probably const_iterable once again after #1992